.github/workflows/manual-publish.yml

```
@@ -64,8 +65,12 @@ jobs:
                   NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
 
             - name: Publish to JSR
-              if: ${{ inputs.package != 'migrate-config' }}
-              run: npx jsr publish
+              run: |
+                  if [ -f "packages/${{ inputs.package }}/jsr.json" ]; then
```

```suggestion
                  if [ -f "jsr.json" ]; then
```

As this runs in the directory of the package (`working-directory: packages/${{ inputs.package }}`).

---

Ah good call.